diff --git a/node_modules/react-draft-wysiwyg/dist/react-draft-wysiwyg.js b/node_modules/react-draft-wysiwyg/dist/react-draft-wysiwyg.js
index 95c0b10..f9579fe 100644
--- a/node_modules/react-draft-wysiwyg/dist/react-draft-wysiwyg.js
+++ b/node_modules/react-draft-wysiwyg/dist/react-draft-wysiwyg.js
@@ -1,3 +1,5 @@
+
+var window = typeof window != "undefined" ? window : {};
 !function(e,t){"object"==typeof exports&&"object"==typeof module?module.exports=t(require("react"),require("draft-js"),require("immutable")):"function"==typeof define&&define.amd?define(["react","draft-js","immutable"],t):"object"==typeof exports?exports.reactDraftWysiwyg=t(require("react"),require("draft-js"),require("immutable")):e.reactDraftWysiwyg=t(e.react,e["draft-js"],e.immutable)}(window,function(n,o,r){return c={},i.m=a=[function(e,t,n){e.exports=n(9)()},function(e,t){e.exports=n},function(e,t,n){var o;
 /*!
   Copyright (c) 2017 Jed Watson.
